src/main/java/com/raisetech/servicepractice/controller/request/StoragePlaceRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.raisetech.servicepractice.controller.request;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+
+import java.time.LocalDate;
+
+public class StoragePlaceRequest {
+    @NotNull
+    private LocalDate storage_date;
```

Comment:
変数はキャメルケースで定義してください。
```suggestion
    private LocalDate storageDate;
```

Reply:
OK🙆‍♂️